src/inventory.h

```diff
@@ -108,15 +108,19 @@ struct ItemStack
 	}
 
 	// Get tool digging properties, or those of the hand if not a tool
-	const ToolCapabilities& getToolCapabilities(
+	ToolCapabilities getToolCapabilities(
```

Comment:
And then I wouldn't need to copy because I could give a reference to that thing. Except I'm not sure if it could still be a const reference, how does that work?